backend/app/controllers/spree/admin/products_controller.rb

```diff
@@ -126,10 +140,20 @@ def variant_stock_includes
         [:images, stock_items: :stock_location, option_values: :option_type]
       end
 
-
       def variant_scope
         @product.variants
       end
+
+      def updating_variant_property_rules?
+        params[:product][:variant_property_rules_attributes].present?
+      end
+
+      def variant_property_rules_for_option_value_ids(current_rule_id, option_value_ids)
```

Comment:
Good catch, must have created it when spiking and forgot to remove it. Just pushed up another commit removing it.